gslib/cred_types.py

```diff
@@ -28,3 +28,4 @@ class CredTypes(object):
   GCE = "GCE"
   DEVSHELL = "Google Developer Shell"
   EXTERNAL_ACCOUNT = "External Account"
+  EXTERNAL_ACCOUNT_AUTHORIZED_USER = "External Account Authorized User"
```

Comment:
We are creating a headful flow for external account users.  When a credential is created using the oauth flow for third party credentials, they are authorized users for external accounts.  I can send you the design document if you're interested

Reply:
I mostly just want to know what "headful" is hahaha

Maybe you should create an internal glossary entry

Reply:
The "Head" is the brains that knows whether or not you're who you're say you are.  The other flows are headless, because the actual authentication is done outside of our control, and we just get a token to prove authenticity.  Here the authorization will happen within the OAuth flow under our control, so it is a headful flow